test/integration/expression_is_true.dbtspec

```diff
@@ -177,42 +111,3 @@ sequences:
           - dbt_utils_expression_is_true_test_table1_attrA_attrB_total
         attributes:
           dbt_utils_expression_is_true_test_table1_attrA_attrB_total.status: fail
-
-  test_check_passes_with_condition:
-    project: project_for_check_passes_with_condition
-    sequence:
-      - type: dbt
-        cmd: deps
-      - type: dbt
-        cmd: seed
-      - type: run_results
-        exists: True
-      - type: dbt
-        cmd: test
-      - type: run_results
-        exists: True
-        length: 1
-        names:
-          - dbt_utils_expression_is_true_test_table1_id_1__attrA_attrB_total
-        attributes:
-          dbt_utils_expression_is_true_test_table1_id_1__attrA_attrB_total.status: pass
-
-  test_check_fails_with_condition:
-    project: project_for_check_fails_with_condition
-    sequence:
-      - type: dbt
-        cmd: deps
-      - type: dbt
-        cmd: seed
-      - type: run_results
-        exists: True
-      - type: dbt
-        cmd: test
-        check: false
-      - type: run_results
-        exists: True
-        length: 1
-        names:
-          - dbt_utils_expression_is_true_test_table1_id_4__attrA_attrB_total
-        attributes:
-          dbt_utils_expression_is_true_test_table1_id_4__attrA_attrB_total.status: fail
```

Comment:
Can you please explain, why these test scenarios are removed?

Reply:
the condition argument is removed from the macro